### DISCUSSION TITLE
Flood DNS Requisite

### DISCUSSION THREAD

**Discussion by thiagodalsoto** [rustdesk/rustdesk#8984] [General]
I'm noticing thousands of DNS resolution requests from all the hosts I'm installing Rustdesk on. Is this due to a configuration error, or is it a bug that I need to report?

As you can see in the printout, there are thousands of type A and AAAA resolution requests
remote.xxx.com.br - address configured in Rustdesk on server
192.168.0.90 - address of the Rustdesk server

![conf_rust](https://github.com/user-attachments/assets/85b53efe-b7c2-43eb-98c1-39af45bb20d1)

![dns_rust-A-AAAA](https://github.com/user-attachments/assets/890f77c9-669b-40ff-a61d-ea70fc4b4061)

**Comment by c-michail** [comment]
I think rustdesk should implement Queries Limit per Minute like DNS Servers have.

**Reply by thiagodalsoto** [reply]
I'm still in doubt, would it be something I can do on my DNS server, or on Rustdesk? Or would this be something to be posted there in failures? Because this behavior shouldn't be like this, imagine a network with about 400 computers, what would it be like?

**Comment by ianalexis** [comment]
Same here, i think this is not acceptable.
Maybe we should create a "Bug" to change it from ~10sec (average i measured) to 5min.
![imagen](https://github.com/user-attachments/assets/4acfaf78-3690-4363-9e91-2c8111ec52d8)

**Reply by ianalexis** [reply]
Found someone whit the same problem.
#5896
